pytest_harvest/tests_raw/test_get_session_results.py

```diff
@@ -210,6 +210,13 @@ def test_synthesis_contains_everything(request):
     assert len(missing) == 0
 
 
+# For some reason, adding a monkeypatch will cause an extra failure for
+# DoctestItem, possibly because it's a setup/teardown
+def test_deal_with_doctest(dummy):
```

Comment:
Can you please extend your comment by explaining what is the `dummy` fixture here ? Maybe it is just a fixture provided by the `monkeypatch` plugin but I did not find any reference to it..